docker-compose.yml

```diff
@@ -58,6 +58,7 @@ services:
       - AZURE_OPENAI_API_KEY=${AZURE_OPENAI_API_KEY}
       - AZURE_OPENAI_API_VERSION=${AZURE_OPENAI_API_VERSION}
       - AZURE_OPENAI_ENDPOINT=${AZURE_OPENAI_ENDPOINT}
+      - MODEL_NAME=${MODEL_NAME}

```

Comment:
I think it's called DEPLOYMENT_NAME. Please see JB Manager and the rest of PwR-Studio and keep the constant the same